src/router.ts

```diff
@@ -274,12 +274,18 @@ const router = createRouter({
       },
     },
   ],
-  scrollBehavior(to /*, from, savedPosition*/) {
-    if (to.hash) {
-      // https://router.vuejs.org/guide/advanced/scroll-behavior.html
+  // https://router.vuejs.org/guide/advanced/scroll-behavior.html
+  scrollBehavior(to, _from, savedPosition) {
+    const defaultScrollOptions: ScrollOptions = { behavior: "smooth" }
+    if (savedPosition) {
+      return {
+        ...savedPosition,
+        ...defaultScrollOptions,
+      }
```

Comment:
```suggestion
      return savedPosition
```

Je trouve que le smooth quand on fait _précédent_ est plutôt déstabilisant.